darts/timeseries.py

```diff
@@ -1803,6 +1804,26 @@ def plot(self,
         plt.legend()
         plt.title(self._xa.name);
 
+    def with_column_renamed(self, col_name: str, col_name_new: str) -> 'TimeSeries':
```

Comment:
Would be nice to accept either single strings, or a sequence of strings here for renaming several columns at once.

Reply:
I wanted to follow pyspark example, where you can't change more than one column but you do  anyway a chain of call like df = (
              df
              .with_column_renamed(a,b)
              .with_column_renamed(c,d)
              .other functions
              ...
         )

but I don't need to follow it stricto sensus.

Reply:
OK, fair enough, let's leave it like that (at least for now) then :)

Reply:
I actually changed it, we can use lists of columns